From 489643230c90d995ad2051c5652f3133cdc8605c Mon Sep 17 00:00:00 2001
From: Carlos Munoz <cmunoz@caviumnetworks.com>
Date: Tue, 26 May 2015 13:26:18 -0700
Subject: [PATCH 121/184] MIPS: OCTEON: Allow SRIO endpoints to complete
 network discovery.

Source: Cavium Networks, Inc.
MR: 00000
Type: Integration
Disposition: Merged from Octeon Tree
ChangeID: d147914e0854da1a8170700330265356f584b255
Description:

Signed-off-by: Carlos Munoz <cmunoz@caviumnetworks.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
[Original patch taken from octeon-linux-kernel-patches-SDK-3.1.2-release]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/cavium-octeon/octeon-rapidio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/cavium-octeon/octeon-rapidio.c b/arch/mips/cavium-octeon/octeon-rapidio.c
index 600987d2..82ad2a6 100644
--- a/arch/mips/cavium-octeon/octeon-rapidio.c
+++ b/arch/mips/cavium-octeon/octeon-rapidio.c
@@ -934,6 +934,7 @@ static void octeon_srio_ports_init(int srio_port)
 	sport->mport.iores.flags = IORESOURCE_MEM;
 	sport->mport.iores.name = "SRIO CSRs";
 	sport->mport.phy_type = RIO_PHY_SERIAL;
+	sport->mport.phys_efptr = 0x100;
 
 	INIT_LIST_HEAD(&sport->mport.dbells);
 	rio_init_dbell_res(&sport->mport.riores[RIO_DOORBELL_RESOURCE], 0,
-- 
1.9.1

